ambry-router/src/test/java/com/github/ambry/router/GetBlobOperationTest.java

```diff
@@ -897,6 +901,39 @@ public void testOrigDcUnavailability() throws Exception {
     getErrorCodeChecker.testAndAssert(RouterErrorCode.AmbryUnavailable);
   }
 
+  /**
+   * Test that if local DC has less than {@code RouterConfig#routerGetOperationMinLocalReplicaCountToPrioritizeLocal}
+   * replicas available, then we will prioritize remote DC.
+   * @throws Exception
+   */
+  @Test
+  public void testGetGoesToRemoteIfLocalDcHasFewAvailableReplicas() throws Exception {
+    doPut();
+    Properties props = getDefaultNonBlockingRouterProperties(true);
+    props.setProperty(RouterConfig.ROUTER_GET_ELIGIBLE_REPLICAS_BY_STATE_ENABLED, "true");
+    props.setProperty(RouterConfig.ROUTER_GET_BLOB_RETRY_LIMIT_COUNT, "10");
+    props.setProperty(RouterConfig.ROUTER_GET_BLOB_RETRY_LIMIT_IN_SEC, "10");
```

Comment:
nit: this confusing, let's not change any retry related configuration.

Reply:
fixed.